apps/discord-bot/src/google-analytic/google-analytic-extended.ts

```diff
@@ -0,0 +1,53 @@
+import { GoogleAnalyticData } from './google-analytic'
+
+export class GoogleAnalyticDataExtended extends GoogleAnalyticData {
```

Comment:
This is for reusable of class, since GoogleAnalyticsData is core system of google analytics which directly use google analytics api, however GoogleAnalyticsDataExtended is only a set of command that a shortcut of calling core analytics command.

 So the main reason is to separate between the necessary one and the shortcut one, however any other solution is accept if you have recommended.

Reply:
There's literally no reason to split up the code into an "Extended" version, when the subclass is already our code in our repo and you're always going to use the extended one. This is way harder to read imo. Put it in the same class.

In fact, the only `core method` that interfaces directly with the google analytics api is the `getMetrics` method, which it literally just an http client.